gui/qt/main_window.py

```diff
@@ -649,7 +649,7 @@ def format_amount_and_units(self, amount):
         return text
 
     def format_fee_rate(self, fee_rate):
-        return format_satoshis(fee_rate/1000, False, self.num_zeros, 0, False)  + ' sat/byte'
+        return format_satoshis(fee_rate/1000, False, self.num_zeros, 1, False)  + ' sat/byte'
```

Comment:
This is not doing what you think it does. You are effectively changing units here.
For example, instead of `6. sat/byte`, it will return `0.6 sat/byte`.

Reply:
Ah yeah, thanks.